examples/hybrid/staggered_nonhydrostatic_model.jl

```diff
@@ -453,84 +532,41 @@ function default_remaining_tendency_special!(Yₜ, Y, p, t)
         end
     end
 end
-function default_remaining_tendency_generic!(Yₜ, Y, p, t)
+
+function explicit_vertical_advection_tendency_generic!(Yₜ, Y, p, t)
```

Comment:
what is the different between generic function and special function?

Reply:
Generic is unthreaded, special is threaded (we should probably change the name)

Reply:
Yeah it should be documented